kiosk/src/Components/EnterHighScore.tsx

```diff
@@ -44,110 +100,56 @@ const EnterHighScore: React.FC<IProps> = ({ kiosk }) => {
     const renderInitials = (): JSX.Element[] => {
         const elements = [];
 
-        console.log("Initial index: " + initialIndexRef.value);
-
         for (let lcv = 0; lcv < configData.HighScoreInitialsLength; lcv++) {
             const thisIndex = lcv;
             elements.push(
-                <HighScoreInitial kiosk={kiosk} key={lcv} isSelected={thisIndex === initialIndex}
+                <HighScoreInitial kiosk={kiosk} key={lcv} isSelected={thisIndex === timesAPressed}
                     onCharacterChanged={character => updateInitial(thisIndex, character[0])} />
             );
         }
 
         return elements;
     }
 
-    const previousInitial = () => {
-        initialIndexRef.value = (initialIndexRef.value + configData.HighScoreInitialsLength - 1) % configData.HighScoreInitialsLength;
-        setInitialIndex(initialIndexRef.value);
-    }
-
-    const nextInitial = () => {
-        initialIndexRef.value = (initialIndexRef.value + 1) % configData.HighScoreInitialsLength;
-        setInitialIndex(initialIndexRef.value);
+    const renderList = (highScores: HighScore[]): JSX.Element[] => {
+        return highScores.map((highScore, i) => {
+            return (
+                <li>
```

Comment:
this needs a key so react can tell when it has changed, if you check console when there are multiple highscores there should be a warning with more info